app/Helpers/Helper.php

```diff
@@ -811,11 +811,13 @@ public static function checkLowInventory()
             }
         }
 
-        foreach ($asset_models as $asset_model){
 
-            $asset = new Asset();
-            $total_owned = $asset->where('model_id', '=', $asset_model->id)->count();
-            $avail = $asset->where('model_id', '=', $asset_model->id)->whereNull('assigned_to')->count();
+
+
+        foreach ($asset_models as $asset_model) {
+
+            $total_owned = $asset_model->assets->count();
+            $avail =  $asset_model->assets->whereNull('assigned_to')->count();
```

Comment:
I think this is going to operate on the *collection* of assets, which we might not want? (Though we *do* already have it loaded up). I think `$asset_model->assets()->whereNull( .... ` might be faster as it won't need the assets loaded up in memory, it will just do a `SELECT COUNT(*) FROM assets WHERE model_id=$blah AND assigned_to IS NULL` which could be a skosh faster?